fees/cian-yieldlayer/index.ts

```diff
@@ -0,0 +1,116 @@
+import { Adapter, FetchOptions } from "../../adapters/types"
+import { CHAIN } from "../../helpers/chains"
+import * as sdk from "@defillama/sdk";
+
+//cian yield layer doesn't have factory, so we hardcode the vaults
+const cianVaults = {
+    [CHAIN.ETHEREUM]: {
+        // "ylstETH Scroll Vault": "0x3498fDed9C88Ae83b3BC6a302108F2da408e613b", this one doesn't have vaultParams and no fee getter, but curiously the website shows 10% fee 
```

Comment:
can you take the list from here? https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/cian-yl/index.js#L36